src/main/java/service/FileReader.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface FileReader {
+    List<String> read(String filePath) throws IOException;
```

Comment:
```suggestion
    List<String> read(String filePath);
```
Don’t  throw exception in the method signature